RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -611,6 +606,52 @@ void consumeStuff()
 	}
 }
 
+boolean consumeFortune()
+{
+	if (contains_text(get_counters("Fortune Cookie", 0, 200), "Fortune Cookie"))
+	{
+		return false;
+	}
+
+	// Don't get lucky numbers for the first semi-rare if we still need to adventure in the outskirts
+	if (my_turncount() < 80 && (internalQuestStatus("questL05Goblin") < 1 && item_amount($item[Knob Goblin encryption key]) < 1))
+	{
+		return false;
+	}
+
+	if (my_level() > 12)
+	{
+		return false;
+	}
+
+	// Try to consume a Lucky Lindy
+	if (inebriety_left() > 0 && auto_is_valid($item[Lucky Lindy]) && my_meat() >= npc_price($item[Lucky Lindy]))
+	{
+		if (autoDrink(1, $item[Lucky Lindy]))
+		{
+			return true;
+		}
+	}
+	
+	// Try to consume a Fortune Cookie
+	if (fullness_left() > 0 && my_meat() >= npc_price($item[Fortune Cookie]))
+	{
+		// Don't eat a fortune cookie if a spaghetti breakfast is still consumable
+		if (auto_is_valid($item[Spaghetti Breakfast]) && item_amount($item[Spaghetti Breakfast]) > 0 && my_fullness() == 0 && my_level() >= 10)
```

Comment:
rather than return false. I would suggest eating Spaghetti Breakfast and then eat the fortune cookie.